### PR TITLE
Feature c

### PR DESCRIPTION
**Pull request by koutlm**

### CODE REVIEW

File: HelloWorld.java [hunk 1]
```diff
@@ -1,6 +1,16 @@
 public class HelloWorld {
 
     public static void main(String[] args) {
-        System.out.println("Hello World");
+        int a = 5;
+        int b = 2;
+        System.out.println(a + "+" + b + "=" + add(a,b));
+    }
+
+    private static int add(int a, int b) {
```
Comment (mkoutli) [inline HelloWorld.java:9]: nice work

File: HelloWorld.java [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+        return a+b;
+    }
+
+    private int add5 (int a) {
+        return a + 5;
```
Comment (mkoutli) [inline HelloWorld.java:14]:
```suggestion
        return a + 10;
```

**Review by mkoutli — CHANGES_REQUESTED** [review]
please also add a subtract method

**Comment by koutlm** [comment]
Close

**Pull request closed without merging**